livestream-compose-sample/src/main/java/io/getstream/livestream/compose/LiveStreamMessageList.kt

```diff
@@ -0,0 +1,20 @@
+package io.getstream.livestream.compose
+
+import androidx.compose.foundation.layout.fillMaxHeight
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import io.getstream.chat.android.compose.ui.messages.list.MessageList
+import io.getstream.chat.android.compose.viewmodel.messages.MessageListViewModel
+
+@Composable
+fun LiveStreamMessageList(messageListViewModel: MessageListViewModel) {
```

Comment:
I don't think we need this component as it's very simple

Reply:
Yeah fair point